README.md

```diff
@@ -32,11 +32,12 @@ Where additional sophistication with AVSs come into the picture:
 2. [Foundry](https://getfoundry.sh/)
 3. [Docker](https://www.docker.com/get-started/)
 
+
 Following NodeJS packages:
 1. tcs
 1. ethers
 
-### Steps
+### Typescript Steps
 
 1. Make sure Docker is running
```

Comment:
Let's move this above as part of 3.1

Reply:
have pushed a change . let me know if this what you meant